### PR TITLE
Remove restriction of creating events only on set tribe workouts

### PR DESCRIPTION
**Pull request by haydenk**
This may not be the direction we want to go but we had/have an event this Saturday we cannot add because our workout days are only M,W,F.

Recurring events are still limited to the days set in the tribe configuration.

Also, I had trouble running the tests successfully, both locally and on Travis CI. Perhaps some help?

### CODE REVIEW

**Comment by tonyd256** [comment]
@haydenk Thank you for contributing! Sorry it took me so long to get back to you, I've been traveling a lot. Anyways, I'll get this in!

File: app/components/result-list.js [hunk 1]
```diff
@@ -29,9 +29,9 @@ export default Ember.Component.extend({
           const xtime = x.get('time');
           const ytime = y.get('time');
 
-          if (xtime === 0) return 1;
-          if (ytime === 0) return -1;
-          if (xtime === ytime) return 0;
+          if (xtime === 0) { return 1;}
```
Comment (tonyd256) [inline app/components/result-list.js]: small thing but would you mind fixing this by adding a space after the semi colon to be consistent.

**Comment by tonyd256** [comment]
Also, would you mind squashing out that middle commit `disable checking if day is set in tribe`

File: app/components/result-list.js (hunk 1 above)
Comment (haydenk) [reply app/components/result-list.js]: Ew!  I can't believe that slipped through.

**Comment by haydenk** [comment]
Looks like I have the commits squashed. I have honestly not had a reason to do that before, so I wasn't that familiar but it was seemed fairly straight forward.

**Pull request merged**